recipes/variant-effect-predictor/build.sh

```diff
@@ -29,3 +29,4 @@ rm -rf t/
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/CADD.pm
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/dbNSFP.pm
 wget --no-check-certificate https://raw.githubusercontent.com/konradjk/loftee/master/LoF.pm
```

Comment:
I don't really think so, the ensembl-variation repo only has a specific branch for the latest  version of vep.
the loftee repo does have some version tags, but the last tag was made over a year ago, while production ready changes still get pushed to the master branch